src/jumpstart/utils.ts

```diff
@@ -0,0 +1,104 @@
+import { Contract } from '@celo/connect'
+import { ContractKit, newKitFromWeb3 } from '@celo/contractkit'
+import jumpstartAbi from 'src/abis/WalletJumpStart.json'
+import { DEFAULT_FORNO_URL } from 'src/config'
+import Logger from 'src/utils/Logger'
+import networkConfig from 'src/web3/networkConfig'
+import { getHttpProvider } from 'src/web3/providers'
+import { getContract } from 'src/web3/utils'
+import Web3 from 'web3'
+
+const TAG = 'WalletJumpstart'
+
+export async function jumpstartLinkHandler(privateKey: string, userAddress: string) {
+  const kit = newKitFromWeb3(new Web3(getHttpProvider(DEFAULT_FORNO_URL)))
+
+  kit.connection.addAccount(privateKey)
+  const accounts: string[] = await kit.connection.getAccounts()
+  const publicKey = accounts[0]
+
+  const jumpstart: Contract = await getContract(jumpstartAbi, networkConfig.walletJumpstartAddress)
+
+  await executeClaims(kit, jumpstart, publicKey, userAddress, 'erc20', privateKey)
+  await executeClaims(kit, jumpstart, publicKey, userAddress, 'erc721', privateKey)
+}
+
+async function executeClaims(
+  kit: ContractKit,
+  jumpstart: Contract,
+  beneficiary: string,
+  userAddress: string,
+  assetType: 'erc20' | 'erc721',
+  privateKey: string
+) {
+  let index = 0
+  while (true) {
+    try {
+      const info =
+        assetType === 'erc20'
+          ? await jumpstart.methods.erc20Claims(beneficiary, index).call()
+          : await jumpstart.methods.erc721Claims(beneficiary, index).call()
+      if (info.claimed) {
+        throw new Error('Already claimed')
```

Comment:
is there a reason to `throw` instead of simply a `continue` (maybe hoisting the log message up from below)?

Reply:
Not really, mb, switched to straight `continue`